lib/terjira/issue_cli.rb

```diff
@@ -109,6 +109,12 @@ def assign(*keys)
       show(issue.key_value)
     end
 
+    desc 'search [SUMMARY]', 'Search for an issue by summary'
+    def search(*keys)
+      search_term = client_class.search(summary: keys[0])
```

Comment:
I also just checked, you can also use `jira issue search query\ string` and it will read as one argument.